globals/colours.css

```diff
@@ -9,7 +9,7 @@
   --color-black:        rgb(51,51, 51);
 
   --color-green:        rgb(41,174,155);
-  --color-red:          rgb(252,99,99);
+  --color-red:          var(--color-coral);
```

Comment:
anywhere you want to use `red` you should put coral. 

Otherwise the excuse is everywhere we currently use green and now want to use teal, we'd just change green to equal teal.